RELEASE_NOTES.rst

```diff
@@ -306,8 +306,8 @@ Refactored Internal Utilities
 
 Several core components have been moved to more intuitive or stable locations:
 
-- The ``SecretsMasker`` class has been relocated to ``airflow.utils.secrets_masker``.
-- The ``ObjectStoragePath`` utility previously located under ``airflow.io`` is now available via ``airflow.utils.object_storage_path``.
+- The ``SecretsMasker`` class has been relocated to ``airflow.sdk.execution_time.secrets_masker``.
+- The ``ObjectStoragePath`` utility previously located under ``airflow.io`` is now available via ``airflow.sdk``.
```

Comment:
This fixes only for forward release. If you want to fix also the 3.0.0 docs you will need to raise PR for https://github.com/apache/airflow-site and manually change it there for 3.0.0

Reply:
Thanks, do you mean editing the html here?

https://github.com/apache/airflow-site/blob/1b2592aca04517d0706dd5920ccd4c60fe4d7281/docs-archive/apache-airflow/3.0.0/release_notes.html#L2604-L2605

Reply:
yea